internal/store/kvbacked.go

```diff
@@ -185,6 +185,9 @@ func NewKVStore(cfg Config) (KVStore, error) {
 		config := etcdclientv3.Config{
 			Endpoints: addrs,
 			TLS:       tlsConfig,
+			DialTimeout: time.Second,
+			DialKeepAliveTime:    time.Second,
+			DialKeepAliveTimeout: 500 * time.Millisecond,
```

Comment:
Also these are too small. What about making the timeout to the current context timeout of 5 seconds?

Reply:
I thing you can grow it as you think be the best